chore: update studio-development.md to explain how to hit the cloud studio without local cypress services (#31591)
cypress-io/cypress: guides/studio-development.md

- Clone the `cypress` repo
  - Run `yarn`
  - Run `yarn cypress:open`
 

> Note: When using the `CYPRESS_LOCAL_STUDIO_PATH` environment variable, we bypass our error reporting and instead decide to throw an exception immediately. The purpose of this is to try and weed out any issues by drawing attention to them immediately

Author: ```suggestion
Note: When using the `CYPRESS_LOCAL_STUDIO_PATH` environment variable, we bypass our error reporting and instead decide to throw an exception immediately. The purpose of this is to try and weed out any issues by drawing attention to them immediately during local development.
```

> Note: When using the `CYPRESS_LOCAL_STUDIO_PATH` environment variable, we bypass our error reporting and instead decide to throw an exception immediately. The purpose of this is to try and weed out any issues by drawing attention to them immediately during local development.

Reviewer: ```suggestion
Note: When using the `CYPRESS_LOCAL_STUDIO_PATH` environment variable, we bypass our error reporting and instead throw the exception immediately. The purpose of this is to try and surface issues immediately during local development.
```
